src/components/GradientPicker/index.js

```diff
@@ -29,7 +29,7 @@ const mapPaletteToStops = ({ palette, activeId, width }) => palette.map((color)
 }));
 
 const getPaletteColor = (palette, id) => {
-	const color = palette.find(color => color.id === id);
+	const color = palette.find(color => color.id === id) || palette[0];
```

Comment:
Mind sharing why it's needed? I tried playing with it and couldn't reproduce any issue ( With removing stop color )

Reply:
When externally setting a completely new palette with less stops then the current activeColorStopId the error occurs.

Reply:
I got ya cool 👍